components/newfoldercomponent/ParagraphComponent.vue

```diff
@@ -6,9 +6,16 @@
 
 <script>
 export default {
+    data(){
+        return {
+
+
+        }
+    },
 
     props: [
-        'paragraphs'
+        'paragraphs',
+        'index'
```

Comment:
Add the following methods:
- totalCharacterCount: it should get the total number of characters that have been typed into the editable div. Make use of `ref` to identify the div
- cursorPosition: It should return the index of the character the cursor is at. So if we have 20 characters in the editable div and the cursor is at character 10, it should return 10. There is a link at the bottom of the doc that shows how to get the count.